src/razor/src/blazorDebug/terminateDebugHandler.ts

```diff
@@ -3,7 +3,7 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import * as psList from 'ps-list';
+import psList from 'ps-list';
```

Comment:
TS then wanted other packages which didn't have a default export to be imported differently.